tools/Makefile.shared

```diff
@@ -105,7 +105,7 @@ install::
 clean::
 	rm -f ocamlmklib
 
-ocamlmklibconfig.ml: ../config/Makefile
+ocamlmklibconfig.ml:: ../config/Makefile
```

Comment:
Yes - because of the addition in `tools/Makefile.nt` because I need an extra line at the end of `ocamlmklibconfig.ml` for Windows only. You can't mix `:` and `::` rules. This has the intentional effect of executing the instructions in `Makefile.shared` followed by the appending to that file in `Makefile.nt` (the `clean` targets in many of the `Makefile`s are constructed in the same way).